site/sfguides/src/hex_titanic/titanic.md

```diff
@@ -0,0 +1,192 @@
+
+![](assets/vhol-execute-write.gif)
+
+We will not be needing these cells anyone, and you can delete them if you would like.
+
+## Create a Snowpark Session
+Duration: 3
+
+Now, we can connect to our Snowflake connection that we imported earlier. To do this head over to the data sources tab on the left control panel to find your Snowflake connection. If you hover your mouse over the connection, you'll see a `query` option appear at the bottom. This will allow us to query this connection in an SQL cell, however, what we want to do is create a Snowpark session using this connection. Click on the dropdown next to the `query` button and select `get Snowpark session`. What this will do is create a new cell for us with all the code needed to spin up a Snowpark session.
+
+
+*The cell created by this button will be positioned under the cell that is currently selected. It is best practice to have this new cell be placed right below package import cell.*
+
+
+![](assets/vhol-Snowpark.gif)
+
+Add the following two lines at the end of the cell to let Snowpark know which schema and database we want to use throughout the project.
+
+```python
+session.use_database(database='PC_HEX_DB')
+session.use_schema("PC_HEX_DB.PUBLIC")
+```
+
+In this cell, we reference our Snowpark session with the variable `session` which is the name assigned by default. Your cell should look like this:
+
+![](assets/snowpark_session.png)
+
+## Hex Project
+Duration: 45
+
+The rest of the lab can entirely be followed step-by-step from the Hex project. Any subsequent steps and explanation will be found in the Hex project itself.
+
+Key sections of the Hex project include:
+- Getting Started
+- Data Exploration
+- Preprocessing and Feature Engineering
+- Model Training
+- Registry
+- Model Inference via a Hex app
+
+
+## Sharing our results
+Duration: 3
+
+Now that we've completed going through our project, how can we share this project with others? To do this we can head over to the app side of things. At the top of your project, you'll see a button showing if you're in the logic or app view. By switching over to the app side of things, we can start arranging our cells and adding/removing them as we see fit. Check out the [Hex docs](https://learn.hex.tech/docs/build-apps/app-builder) to learn about how you can effectively use the app builder.
+
+A pre-built layout has been designed but feel free to customize and add any additional elements!
+
+Once you are happy with the layout, we can go a head and publish. Publishing is like taking a snapshot of the current state of our app and publishing it as an interactive web app that anyone can see. To publish, hit the green publish button in the top right corner. Once that's done, hit the  share button next to publish, and select `Publish to web`. Whoever you share the link with will be able to see the live version of your app!
+
+
+## Conclusion
```

Comment:
Make the layout and structure similar to https://quickstarts.snowflake.com/guide/getting_started_with_dataengineering_ml_using_snowpark_python/index.html#0